src/MICore/Transports/StreamTransport.cs

```diff
@@ -207,9 +194,34 @@ public bool IsClosed
             get { return _bQuit; }
         }
 
+        public virtual int DebuggerPid
```

Comment:
I can see why. So the exception will be thrown explicitly in the subclasses if the property doesn't make sense, correct?